docs/telemetry.md

```diff
@@ -0,0 +1,206 @@
+# Telemetry labs
+
+## gNMIc Prometheus Grafana Lab
+
+### Lab Topology
+
+![lab topology](assets/img/aclabs-telemetrylab-A.png)
+
+### Demo Details
+
+!!! Success "Container Requirements"
+
+    CPUs: 8
+    Memory: 32 GB
+    Storage: 64 GB
+
+### CPU Architechture
+
+- Only x86 CPU architecture is supported!
+- ARM support coming soon!
+
+### How To Run The Workshop
+
+???+ Tip "Wait until cEOS-lab image will be imported"
+
+    You can confirm if image was imported correctly with `docker image ls`.
+    In some rare cases the cEOS-lab image may fail to be downloaded.
+    For example due to incorrecte token or other API failure on arista.com.
+    In that case you can upload the image manually by right-clicking the Explorer tab of the VSCode interface.
+
+To run the labs in this workshop, you can use one of the following options:
+
+- Build you own Ubuntu VM from scratch. (Recommended!)
+  - This option will allow you to experience the entire cLab enviroment build process from the very start, without any pre-installed dependancies
+  - Requirements
+    - Ubuntu LTS 22.04 or later
+    - 32 GB RAM and 8 vCPUs
+- Start the [Github Codespace](https://codespaces.new/noredistribution/aclabs/tree/main?quickstart=1&devcontainer_path=.devcontainer%2Fgnmic-prometheus-grafana%2Fdevcontainer.json) from this repository. (Fastest!)
+- Open the lab dev container locally on your laptop with Docker Desktop or sever with Docker CE:
+  - download one of the prepackage AVD+Python+Containerlab containers from [aclabs packages](https://github.com/aristanetworks/aclabs/pkgs/container/aclabs%2Flab-base)
+  - when running the lab locally, you must set all required environment variables on your machine
+
+### Run it in Codespaces
+
+[Start :octicons-play-16: gNMIc-Prometheus-Grafana Lab :octicons-mark-github-24:](https://codespaces.new/noredistribution/aclabs/tree/main?quickstart=1&devcontainer_path=.devcontainer%2Fgnmic-prometheus-grafana%2Fdevcontainer.json){ .md-button .md-button--primary target=_blank}
+
+!!! Note
+
+    The ARTOKEN variable should be set for the scripts to be able to download the cEOS-lab image. Please set this before clicking on `Create new codespace` as mentioned in the [Quickstart](quickstart.md#starting-the-lab)
+
+Once the web based vscode starts the containerlab topology should be automatically deployed via the `postCreate.sh` script, and once the grafana container is ready, Codespaces will send a notification and reveal the `Open in Browser` button.
+
+![codespace-grafana](assets/img/codespace-grafana.png)
+
+To access Grafana use the `arista/arista` credentials.
+
+To add configurations to the switches, such as configuring EVPN, we can use AVD for instance (This will be needed for the L3 Telemetry dashboard):
+
+  ```bash
+  ansible-playbook playbooks/fabric-deploy-config.yaml -i inventory.yaml
+  ```
+
+### How To Run The Lab on Your Machine
```

Comment:
We can keen everything below this header for now. But definitely this has to be moved out of the lab guide over time. Somewhere to `Home` section.